### ISSUE TITLE
Spinner (loading animation) update

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/ibm-dotcom-library#206]
_Wonil-Suh1 created the following on Mar 09:_  

 ## User story

As a user of IBM.com I need to know that the system is loading something, in a consistent way across web and products so that I can work better and be happier.

## Notes

The digital loading spinner (https://www.ibm.com/standards/web/components/loading-spinner/) is different from Carbon spinner (https://next.carbondesignsystem.com/components/loading/code). Let's combine the style so that it's consistent.

## Acceptance criteria

- updated design and animation specs
- specs are communicated to Carbon team
- The digital design spinners are updated to the specs
 

_Original issue: https://github.ibm.com/webstandards/digital-design/issues/625_

**Comment by jeffchew** [comment]
_On Mar 11, chsanche commented:_
**Note:** Once the component update is done, please create an issue for the website to update the animation as well. Thank you.

**Comment by jeffchew** [comment]
_On Mar 11, Olivia-Flory commented:_
@Wonil-Suh1 are we wanting to update the current design for Northstar v18 and v19 alpha?

Or are you suggesting we propose that Carbon updates their spinner to look like the one we have?

**Comment by jeffchew** [comment]
_On Mar 25, chsanche commented:_
@Wonil-Suh1 👆👆👆

**Comment by jeffchew** [comment]
_On Mar 25, Wonil-Suh1 commented:_
I say we update to match Carbon. I'm looking the speed and line weight.

**Comment by jeffchew** [comment]
_On Mar 26, Olivia-Flory commented:_
Should we close this issue if we agreed to freezing v18/v19a code?

**Comment by JennySanchez** [comment]
James and I looked into this and this issue is no longer relevant and will be closed.
